variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -141,20 +147,29 @@ public int getLength() {
     public VariantType getType() {
         if (this.alternate.equals(".")) {
             return VariantType.NO_ALTERNATE;
-        } else if (reference.length() == alternate.length()) {
+        } else if (this.reference.startsWith("<") && this.reference.endsWith(">")
+                   || this.alternate.startsWith("<") && this.alternate.endsWith(">")) {
```

Comment:
Not an immediate necessity but something to ponder: if the symbolic allele is "<>" this will return the type SEQUENCE_ALTERATION.  Looks like this might not be a big deal if the eventual write to a VCF will error out since HTSGET will consider this an invalid alternate. But we can make it a TODO to make this type deduction more robust in the future by adding a ">2" length check.

Reply:
small correction: I was talking about htslib, when writing variants into a VCF

Reply:
also, I confirm that the validator will complain on ALT=`<>`